docs/antora.yml

```diff
@@ -1,6 +1,6 @@
 name: jte
 title: Jenkins Templating Engine
-version: 2.0-alpha2
+version: 2.0.1
```

Comment:
can we revert this change?

Reply:
what should be the antora version? 2.0.3? or 2.0.4 or something else